app/views/govuk_publishing_components/components/docs/image_card.yml

```diff
@@ -205,8 +205,8 @@ examples:
       <div class="govuk-!-width-full">
         <%= component %>
       </div>
-  two_thirds_column:
-    description: This variant is used for the featured section on the homepage, the aspect ratio used is 1:1
+  two_thirds:
+    description: This variant is used for the featured section on the homepage. The aspect ratio used is 1:1 and the width of the image is constrained to 90px wide.
```

Comment:
A tiny thing, but should this say `95px wide`?

Reply:
Thanks Hanna, good spot, I've just updated this, it should actually read `80px wide`, as the flex-basis for the image card here is set to 95px, padding is 15px, leaving 80px for the image. I've also updated the related comment in the SCSS file around this.